### ISSUE TITLE
CUDA implementation request

### ISSUE THREAD

**Issue by nft30001** [kanhavishva/keyhunt-ocl#7]
can you please compile this program for CUDA same as KeyHunt-Cuda-2? 
and Thank you so much for you continue awesome work ~!!!

**Comment by studyroom1** [comment]
@kanhavishva is it possible to compile it for cuda?

**Comment by kanhavishva** [comment]
yes it is possible but not directly, need to change a lot of things to make it workable on cuda.

**Comment by kanhavishva** [comment]
can use other programs for cuda, like keyhunt-cuda

**Comment by nft30001** [comment]
@kanhavishva but seems like CUDA and OCL are both different from each other in order to search PKs? aren't they ~~

**Comment by kanhavishva** [comment]
yes, right now keyhunt-cuda and keyhunt-ocl both are different in key searching because both are the modified versions of their different type of parent projects.

if a program is written for both platforms by the same person, then there would be a possibility that both works in the same way,
for example: [BitCrack](https://github.com/brichard19/BitCrack) by [brichard19](https://github.com/brichard19).

**Comment by nft30001** [comment]
@kanhavishva thanks for the explanation is there any possibility we can see CUDA version of this soon?

**Comment by studyroom1** [comment]
@kanhavishva i am not sure but i think you are avoiding to discuss 3000 series issue  :(

**Comment by kanhavishva** [comment]
i'm busy on other things, and don't have rtx3080 too, so can't test in real-time.

@studyroom1 studyroom1 -> https://github.com/kanhavishva/keyhunt-ocl/issues/6#issuecomment-873241648
